documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,78 @@
 
 === Board Support
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The Touch Display is designed to work with all models of Raspberry Pi, however early models without mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-sized stand-offs on the display PCB.
 
-=== Physical Installation
+=== Connecting a Raspberry Pi
 
-The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (ribbon cable) and power (red/black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power wires should be attached to the GND and 5V pins respectively.
+The following image shows how to attach a Raspberry Pi directly to the back of the Touch Display using its standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the Touch Display in a separate chassis if you have one available. The connections will remain the same, though you may need longer cables depending on the chassis you use.
 
-image::images/GPIO_power-500x333.jpg[DSI Display Connections]
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
 
-The other three pins should be left disconnected, unless connecting the display to an original Raspberry Pi 1 Model A or B. See the section on xref:display.adoc#legacy-support[legacy support] for more information on connecting the display to an original Raspberry Pi.
+==== Connecting the display cable
 
-NOTE: An original Raspberry Pi can be easily identified from other models, it is the only model with a 26-pin rather than 40-pin GPIO header connector.
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB so the side with the blue stripe faces the display. Then connect the other end of the ribbon cable to the `DISPLAY` port on the Raspberry Pi. The blue stripe should face away from the Raspberry Pi.
+
+If the ribbon cable isn't fully inserted, or it's not positioned correctly, you will experience issues with the display. You should always double-check this connection when troubleshooting, especially if you don't see anything on your display, or the display is showing a single color.
+
+==== Powering the Touch Display
+
+There are two options available to provide power to the Touch Display: using the Raspberry Pi's GPIO, or a separate micro USB power supply. We recommend using the Raspberry Pi to provide power to the Touch Display whenever possible.
+
+===== Provide power through the Raspberry Pi
```

Comment:
We're nesting fairly deeply here. Is there any way to reorganize this so it's a flatter structure?